norduser/group_monitor.go

```diff
@@ -106,82 +123,139 @@ func getUID(username string) (userIDs, error) {
 	}, nil
 }
 
-func (n *NordvpnGroupMonitor) handleGroupUpdate(currentGroupMembers userSet, newGroupMembers userSet) {
-	for member := range currentGroupMembers {
-		// member not modified, do nothing
-		if _, ok := newGroupMembers[member]; ok {
-			newGroupMembers[member] = true
+func parseSessions(sessions string) []string {
+	re := regexp.MustCompile(`(?m)^(\S+)`)
+	matches := re.FindAllStringSubmatch(sessions, -1)
+	if matches == nil {
+		return []string{}
+	}
+
+	activeUsers := []string{}
+	for _, match := range matches {
+		activeUsers = append(activeUsers, match[1])
+	}
+
+	return activeUsers
+}
+
+func getSessions() []string {
+	cmd := exec.Command("w", "--no-header")
```

Comment:
I would prefer another solution than executing an external binary. We already do too many things by calling external command binaries and parsing their output. This is both resource intensive and error prone, because output formats may differ between systems or versions (for example recent bug with parsing uids in `ps` output which resulted in tray icons spam).